packages/config/config/devices/0x015f/mh-s212.json

```diff
@@ -117,17 +116,17 @@
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
-			"defaultValue": 0,
+			"defaultValue": 1,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true,
 			"options": [
 				{
-					"label": "Toggle Switch without specified on/off position",
+					"label": "Forbid Manual Operation",
 					"value": 0
 				},
 				{
-					"label": "Toggle Switch with specified on/off position",
+					"label": "Allow Manual Operation",
 					"value": 1
 				}
```

Comment:
I don't understand what these options have to do with the label `"Switch 1 - Off Timer"`

Reply:
Yeah, I'm not sure to be honest.   I'm wondering whether it should just be "Disable" and "Enable", which will enable at automatic off timer after pressing Switch 1.  (With the following parameter 6 (Timing Function) specifying the amount of time to turn off?

![image](https://user-images.githubusercontent.com/34757455/114385169-3cbe5780-9bc2-11eb-8dcc-4bbe2cf46cdb.png)

Reply:
I'm thinking that approach makes sense, so I've updated to suit.